thavalon-webapp/src/components/gameComponents/interactions/missionManager.tsx

```diff
@@ -102,50 +173,57 @@ export function MissionManager(props: MissionManagerProps): JSX.Element {
 function MissionCardButtons(props: MissionCardProps): JSX.Element {
     const buttons = new Array<JSX.Element>();
     for (const card of Object.values(MissionCard)) {
+        let className = "mission-button-success";
+        switch (card) {
+            case MissionCard.Fail:
+                className = "mission-button-fail";
+                break;
+            case MissionCard.Reverse:
+                className = "mission-button-reverse";
+                break;
+            case MissionCard.QuestingBeast:
```

Comment:
spacing somehow got messed up in these cases

Reply:
Spacing in the git preview and in the code looks normal to me.